src/commands/selections.ts

```diff
@@ -326,7 +326,13 @@ export function filter(
   defaultInput?: Argument<string>,
   inverse: Argument<boolean> = false,
   interactive: Argument<boolean> = true,
+  setMainIndex: Argument<boolean> = false,
+  count: number = 0,
 ) {
+  if (setMainIndex) {
+    input = "i == " + count
```

Comment:
This line will likely be deleted given my above comment, but FYI lines should end with a semicolon and strict equalities `===` are preferred, except in rare cases.

Reply:
The perils of writing code at night ;)